frontend/src/app/actions/newsletter-form.js

```diff
@@ -32,6 +31,9 @@ function newsletterFormSubscribe(dispatch, email, locale) {
       }
     })
     .catch(() => dispatch(actions.newsletterFormSetFailed()));
+  return {
+    type: 'NEWSLETTER_FORM_SUBSCRIBE'
+  };
```

Comment:
This was a bugfix, I'm not sure what exposed it or introduced it. redux doesn't like `undefined` as an action.